scripts/remove-svc.sh

```diff
@@ -73,4 +73,4 @@ if [ "${runner_plat}" == "linux" ]; then
 fi 
 ${prefix}./svc.sh stop
 ${prefix}./svc.sh uninstall
-${prefix}./config.sh remove --token $REMOVE_TOKEN
+,/config.sh remove --token $REMOVE_TOKEN
```

Comment:
you might want to check how we configure the runner in the [config script](https://github.com/actions/runner/blob/main/scripts/create-latest-svc.sh), and make sure we lines up with that.

Reply:
Good point, in the create script we run `sudo -E -u ${svc_user} ./config.sh` to preserve the environment variables so we don't trip on `RUNNER_ALLOW_RUNASROOT`. I agree it should be consistent.

Is there a reason to run `./config.sh` with `sudo` at all? If there isn't, shouldn't both just be `./config.sh`?